packages/next/src/client/components/error-boundary.tsx

```diff
@@ -4,8 +4,9 @@ import React from 'react'
 
 const styles = {
   error: {
+    // https://github.com/sindresorhus/modern-normalize/blob/main/modern-normalize.css#L38-L52
     fontFamily:
-      '-apple-system, BlinkMacSystemFont, Roboto, "Segoe UI", "Fira Sans", Avenir, "Helvetica Neue", "Lucida Grande", sans-serif',
```

Comment:
Fonts like Fira Sans are not standard system fonts, as they are not included in macOS or Windows by default ([ref](https://web.mit.edu/jmorzins/www/fonts.html)). Here we change it to a more standard reset from `modern-normalize`.